src/mpid/ch4/netmod/ofi/subconfigure.m4

```diff
@@ -298,6 +298,7 @@ AM_COND_IF([BUILD_CH4_NETMOD_OFI],[
         dnl Unset all of these env vars so they don't pollute the libfabric configuration
         PAC_PUSH_ALL_FLAGS()
         PAC_RESET_ALL_FLAGS()
+        CFLAGS="$CFLAGS $VISIBILITY_CFLAGS"
```

Comment:
Unclear why this is needed. Nothing appears to have changed in libfabric from what I can see.